src/main/java/org/pesmypetcare/webservice/entity/medalmanager/MedalEntity.java

```diff
@@ -0,0 +1,43 @@
+package org.pesmypetcare.webservice.entity.medalmanager;
+
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Oriol Catalán
+ */
+@Data
+public class MedalEntity {
+    public static final String NAME = "name";
+    public static final String LEVELS = "levels";
+    public static final String DESCRIPTION = "description";
+    public static final String ICON_LOCATION = "iconLocation";
+    private String name;
+    private List<Double> levels;
+    private String description;
+    private String iconLocation;
+
+
+
+    public MedalEntity() { }
+
+    public MedalEntity(String name, ArrayList<Double> levels, String description) {
+        this.name = name;
+        this.levels = levels;
+        this.description = description;
+        this.iconLocation = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assigning an Object to null is a code smell.  Consider refactoring.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517612)